subsys/net/ip/ipv4_autoconf.c

```diff
@@ -196,7 +196,9 @@ static void ipv4_autoconf_send(struct net_if_ipv4_autoconf *ipv4auto)
 			break;
 		}
 		__fallthrough;
-	case NET_IPV4_AUTOCONF_ANNOUNCE:
+	case NET_IPV4_AUTOCONF_ANNOUNCE:{
```

Comment:
This weird, in terms of code style. Shouldn't the '{ be one line after and indented accordingly? Same for the closing '}'. Now it's all at the same level. 

Perhaps better externalizing this code block into a dedicated inline function.

Reply:
I think that the current approach is correct in terms of coding style (with the exception for missing space before `{`). We only move the opening brace to the line below in function definitions. This approach follows Linux coding style guidelines and is used widely across Zephyr codebase. 

I've followed the second recommendation though, and extracted the code block into a sperate function.